src/version/version.F90

```diff
@@ -16,33 +16,32 @@
 
     MODULE GALAHAD_VERSION
 
-      USE GALAHAD_KINDS_precision
-
       IMPLICIT NONE
 
       PRIVATE
-      PUBLIC :: VERSION
+      PUBLIC :: VERSION_galahad
 
       INTEGER, PARAMETER :: major = 5
```

Comment:
It should be 64-bit integers if we compile GALAHAD with` -DINTEGER_64`.

Reply:
No, these are tiny integers, no need for 64 bit, and there are no real values so do we need _s?

Reply:
Yes these are tiny integers but we could want 64-bit integers for compatibility reasons with other languages like MATLAB.

For the `_s`, you are right, we don't need it.

Reply:
> Yes these are tiny integers but we could want 64-bit integers for compatibility reasons with other languages like MATLAB.
> 
OK, we can do that. On my list (the sun is out, and I am on vacation, so I'll probably
head out to the garden now)